example/src/main/java/io/netty/example/http2/helloworld/client/Http2Client.java

```diff
@@ -77,8 +76,7 @@ public static void main(String[] args) throws Exception {
                     SelectorFailureBehavior.NO_ADVERTISE,
                     // ACCEPT is currently the only mode supported by both OpenSsl and JDK providers.
                     SelectedListenerFailureBehavior.ACCEPT,
-                    SelectedProtocol.HTTP_2.protocolName(),
-                    SelectedProtocol.HTTP_1_1.protocolName()))
+                    "h2", "http/1.1"))
```

Comment:
@nmittler - I wonder if it is better to have a more centralized approach to represent the TLS-ALPN IANA registered extensions are defined.  This would make it more clear when using what are the "supported" or viable options to choose from.  NPN and ALPN definitions agree so these should be consistent.  [Http2CodecUtil](https://github.com/netty/netty/blob/master/codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java#L40) definitions may need to be updated to reference these definitions.

https://github.com/netty/netty/pull/3846#issuecomment-107820799